From 216aed597b3f20692a9c9d74dae79fa73e36c75b Mon Sep 17 00:00:00 2001
From: Andrei Dinu <andrei.adrianx.dinu@intel.com>
Date: Thu, 1 Aug 2013 10:30:26 +0300
Subject: [PATCH] added functionality

 * removed sshd from the checks. When needed it can be added
 in the file.
 * added saving location for checksums.gpg file

Signed-off-by: Andrei Dinu <andrei.adrianx.dinu@intel.com>
---
 conf/buck-security.conf |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/conf/buck-security.conf b/conf/buck-security.conf
index eb88982..f573aa8 100644
--- a/conf/buck-security.conf
+++ b/conf/buck-security.conf
@@ -27,7 +27,6 @@ usermask
 superusers
 services
 firewall
-sshd
 packages_problematic
 checksum
 );
@@ -48,7 +47,7 @@ $checksum_program = "sha256sum";
 $checksum_dir = "/sbin/* /bin/* /usr/sbin/* /usr/bin/*";
 
 
-$checksum_file = "checksums.gpg";
+$checksum_file = "/usr/bin/checksums.gpg";
 
 
 
-- 
1.7.9.5

